automated/linux/igt/igt-chamelium-test.yaml

```diff
@@ -31,6 +30,8 @@ run:
         - cd ./automated/linux/igt
         - git config --global http.sslverify false
         - if [ -n "${TEST_LIST}" ]; then TL="-t ${TEST_LIST}"; fi
+        # ${CHAMELIUM_STATIC_IP} is from LAVA device dictionary
+        - CHAMELIUM_IP=`echo ${CHAMELIUM_STATIC_IP}`
```

Comment:
what is the reason for using 'echo'? Wouldn't simple assignment work? Also, maybe we should simply change the name of the variable in device dictionaries to match the script?